src/services/wallet-service/wallet-service.ts

```diff
@@ -66,7 +74,7 @@ export class WalletService implements IWalletService {
 
     getWallets() {
         return new Promise(resolve => {
-            this.http.get('assets/data/walletData.json').subscribe(data => {
+            this.http.get('http://localhost:3000/wallets').subscribe(data => {
```

Comment:
We must think of a way to dynamically set the host in the build proces. Or: simply write the url as `/wallets`.